src/renderer/pages/Apps.tsx

```diff
@@ -148,6 +150,12 @@ export default function Apps() {
       }
     }
 
+    if (appConfig?.stableNodeVersion !== nodeConfigRelease) {
```

Comment:
not a snackbar, ideally we want a warning logo (maybe you can get one from fa icons) at the right most side. if you hover on it, you should see the warning message.

it might also help to create a reusable hover component here, we can use it at other places as well